DevelopGitHub/GitHubExtension/GithubExtension.Extensions/UseOwinContextInjectorExtension.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Owin;
+using SimpleInjector;
+using SimpleInjector.Extensions.ExecutionContextScoping;
+
+namespace GithubExtension.Extensions
+{
+    public static class UseOwinContextInjectorExtension
+    {
+        public static void UseSimpleInjectorContext(this IAppBuilder app, Container container)
+        {
+            // Create an OWIN middleware to create an execution context scope
+            app.Use(async (context, next) =>
```

Comment:
Use this extension method for correct work owin with simple injector, according to this tutorial http://simpleinjector.readthedocs.org/en/latest/owinintegration.html